### DISCUSSION TITLE
Select multiple columns

### DISCUSSION THREAD

**Discussion by age3rcm** [opentoonz/opentoonz#6978] [Ideas]
there is no known method to select multiple colums to animate cut-out style
this is crucial to cut-out animation workflow

select multiple columns possible

(
![Animate_NHxK4MNNG8](https://user-images.githubusercontent.com/36406840/117212202-6f79fb00-add0-11eb-83d3-751c6818daa2.png)
example in adobe animate)

**Comment by DarrenTAnims** [comment]
Can you give an example of when and how you need to move multiple columns?
Ordinarily, when I've used cut out, I rotate individual columns. And if I need to move the character, I move the root column or add a root parent column just for moving.

**Comment by age3rcm** [comment]
considering there are parts not connected on the chain, you could use the multiple selection to move
for example: selecting all legs to match body position
while you have the legs separated from the torso chain

**Comment by age3rcm** [comment]
also when dealing with props
you could select many of them and move them all at once
you might have been in a hurry and did not bother to create a proper rig tree with adequate parenting, and each prop is its own node

**Comment by RodneyBaker** [comment]
> there is no known method to select multiple colums to animate cut-out style

The standard  way to select and manipulate multiple columns in Opentoonz is to group the columns as a subxsheet.

Parenting is another way to 'select' multiple columns.
There are a number of ways to do this; Pegs, null columns, parent columns...

We can also use Expressions to connect columns.

Over in Tahoma2D fork they do have Folders that can be used to organize column but as of this date the folder cannto be animated; although the Function Editor does expose channels that suggest they can be manipulated.  The assumption is that animating Folders of columns will be possible at some point in the future.

**Comment by RodneyBaker** [comment]
Will move this to Discussion to get better clarification.
We can currently select multiple columns in the xsheet but that isn't the request here.
